### DISCUSSION TITLE
Firestick.

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#390] [General]
Just got as a present this streaming device and have absolutely no idea what to do with it.
Any tips?

**Comment by hernessteve** [comment]
Not much to add here. I would only recommend installing a private internet access on your firestick too and choose a good vpn service for streaming purposes - [https://www.firesticktricks.com/install-private-internet-access-on-firestick.html](https://www.firesticktricks.com/install-private-internet-access-on-firestick.html). Firestick tricks website has a lot of useful tips and instructions and i am sure you will find everything you want to know about the firestick there.
